react-delegationdashboard/src/components/Layout/index.tsx

```diff
@@ -16,6 +16,16 @@ import React from 'react';
 import { calculateAPR } from './APRCalculation';
 import Footer from './Footer';
 import Navbar from './Navbar';
+import axios from 'axios';
+
+const getStakingSCBalance = async (): Promise<string> => {
+  const result = await axios.get(`${network.apiAddress}/accounts/${auctionContract}`);
```

Comment:
can you use `doGet` from erdjs.ProxyProvider ?

Reply:
for the moment doGet is private so we can not use it, I'll create a PR this days to update the library and change doGet